src/data/graphql.tsx

```diff
@@ -4435,8 +4480,10 @@ export type LanguageWhereUniqueInput = {
 }
 
 export enum Locale {
-  EnSe = 'en_SE',
   SvSe = 'sv_SE',
+  EnSe = 'en_SE',
+  NbNo = 'nb_NO',
```

Comment:
nb? 🤔 
```suggestion
  NbNo = 'nb_NO',
```

Reply:
That's "norska bokmål" I'm guessing :)

Reply:
I mean this is from the graphql schem, but i'm assuming `nb` is for `Norskt Bokmål` which is the official written norwegian language :)